e2e/js_image_oci/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "image",
     "dependencies": {
-        "chalk": "^5.2.0"
+        "chalk": "^4.0.0"
```

Comment:
5.2 actually causes an error

I think renovate probably updated this but because there was no pnpm-workspace.yaml it updated the root pnpm-lock.yaml and had no effect on this e2e test. I added the pnpm-workspace in this PR

Reply:
Yeah, chalk 5 is esm only and that breaks things if I recall